ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/fulu/helpers/MiscHelpersFulu.java

```diff
@@ -125,9 +128,21 @@ public int getMaxBlobsPerBlock(final UInt64 epoch) {
         blobSchedule.stream()
             .filter(blobSchedule -> blobSchedule.epoch().isLessThanOrEqualTo(epoch))
             .max(Comparator.comparing(BlobSchedule::epoch));
-    return maybeSchedule.isPresent()
-        ? maybeSchedule.get().maxBlobsPerBlock()
-        : blobSchedule.getFirst().maxBlobsPerBlock();
+
+    final int maxBlobs =
+        maybeSchedule
+            .map(BlobSchedule::maxBlobsPerBlock)
+            .orElseGet(() -> blobSchedule.getFirst().maxBlobsPerBlock());
+    LOG.debug("Max blobs at epoch {} found to be {}", epoch, maxBlobs);
```

Comment:
We will log it several times per each slot, maybe debug is too spammy for this?

Reply:
well, i'd say if its getting super chatty we should probably be caching it somewhere... for minimal mode i was seeing it once per block

Reply:
for this PR that's kinda good if you've got debug enabled and you see it a lot and know something's not perfect i think.... with a tiny list the overhead is small but over time having tons of calls will be less cheap